pandas/core/indexes/interval.py

```diff
@@ -865,6 +863,22 @@ def _convert_list_indexer(self, keyarr):
 
     # --------------------------------------------------------------------
 
+    @cache_readonly
+    def left(self) -> Index:
+        return Index(self._values.left, copy=False)
+
+    @cache_readonly
+    def right(self) -> Index:
+        return Index(self._values.right, copy=False)
+
+    @cache_readonly
+    def mid(self):
+        return Index(self._data.mid, copy=False)
```

Comment:
Is there a reason you use `_values` for left/right, but `_data` for mid/length ?

(I suppose both are fine, since they are aliases, but should try to be consistent in which one is used here?)

Reply:
no, will standardize